CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+<a name="1.15.5"></a>
+## <small>1.15.5 (2018-12-05)</small>
+
+* changelog ([d2b636e](https://github.com/bigcommerce/stencil-cli/commit/d2b636e))
```

Comment:
Why is it not linking the PR in the changelog ?

Reply:
see https://github.com/conventional-changelog/standard-version/blob/master/CHANGELOG.md